doc/api/test.md

```diff
@@ -733,7 +733,7 @@ changes:
     For subtests, it will be `Infinity` tests in parallel.
     If falsy, it would only run one test at a time.
     If unspecified, subtests inherit this value from their parent.
-    **Default:** `false`.
+    **Default:** `1`.
```

Comment:
`false` and `1` are strictly equivalent, I think it's better to document `false` as I expect most users to only use a boolean value for this field. No strong feeling though.

Reply:
I can revert this although `test` and `context.text` will have different default values, unless that's not an issue?

Reply:
Consistency would be better, so I'd say we should put `false` for both, but also I'm not feeling strongly either way.

Reply:
+1 for false

Reply:
Sure, I've pushed an update that reverts this change but updates the `test` concurrency option so that the two are consistent.